helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -433,6 +432,92 @@ public void removeDeploymentGroup(final String name) throws DeploymentGroupDoesN
     }
   }
 
+  public List<RollingOperation> getRollingOperations(final String groupName)
```

Comment:
For now yes (I didn't know about `@VisibleForTesting`), but I expect we'll want to expose it in future so we can add functionality to list recent rolling ops for a DG via the API/CLI.